src/java/org/apache/cassandra/service/accord/AccordJournal.java

```diff
@@ -716,6 +716,7 @@ public void close()
 
                 ++cur;
             }
+            return true;
```

Comment:
should we return false on corruption?

Reply:
This logic is essentially disabled for now as we discussed, I will introduce it more fully in a follow up patch.

Reply:
I can not recall any conversation about this logic being disabled. Searched messages quickly and couldn't find anything either. Maybe I misunderstood when it got mentioned.